TestStream.java

```diff
@@ -1,2 +1,33 @@
+import java.util.Arrays;
+import java.util.Comparator;
+import java.util.List;
+import java.util.stream.IntStream;
+import java.util.stream.Stream;
+
 public class TestStream {
+    public static void main(String[] args) {
+//王を含むものだけ表示
+        List<String> taiyoukeiList = List.of("水星", "金星", "地球", "火星", "木星", "土星", "天王星", "海王星");
```

Comment:
個人的にはコメントは下記のようにコードにインデントを合わせるのが好みです。
```suggestion
        //王を含むものだけ表示
        List<String> taiyoukeiList = List.of("水星", "金星", "地球", "火星", "木星", "土星", "天王星", "海王星");
```

下記はSpring Bootのソースコードですが、こういうイメージです。
https://github.com/spring-projects/spring-boot/blob/26cec61f3209e487cc2ff6deb6a2e571804267a8/spring-boot-project/spring-boot/src/main/java/org/springframework/boot/context/properties/ConfigurationPropertiesReflectionHintsProcessor.java#L168-L173

Reply:
ハジに揃えたた方がいいのかな？と思っていましたが、確かにこの方が見えやすいですね。次からは揃えます！